core/error_api.php

```diff
@@ -403,7 +403,7 @@ function error_handler( $p_type, $p_error, $p_file, $p_line ) {
 					if( defined( 'DELAY_INLINE_ERROR_REPORTING')) {
 						error_log_delayed( $t_error_type . ': ' . $t_error_description );
 					} else {
-						echo '<div class="alert alert-warning">', $t_error_type, ': ', $t_error_description, '</div>';
+						echo '<div class="alert alert-warning" style="position: relative; top: 4em;">', $t_error_type, ': ', $t_error_description, '</div>';
```

Comment:
This is fine when the inline warning appears at the top the page, but the `position: relative` messes up display when it's at the bottom as the alert(s) are shifted down and overlap the bottom elements. For example:

![image](https://github.com/user-attachments/assets/4c530558-b230-4145-9284-741c80a3235a)
![image](https://github.com/user-attachments/assets/ae0d6eb1-f1d0-43cf-a45a-94dd5d6dad89)

Reply:
Indeed. There are other ways we can try to solve the problem:
- Add the style `z-index: 10000`, then the element will be on top and overlap the others.
- Define a default constant: `define( 'DELAY_INLINE_ERROR_REPORTING', true );` so that all errors go to the delayed messages panel.
- Add a mechanism to keep track of when the page title is printed, and elements before the `navbar` should be printed with `position: relative`, and after - without.